aiidalab_widgets_base/computers.py

```diff
@@ -668,13 +668,18 @@ class ComputerDropdown(ipw.VBox):
     selected_computer = Union([Unicode(), Instance(Computer)], allow_none=True)
     computers = Dict(allow_none=True)
 
-    def __init__(self, text='Select computer:', path_to_root='../', **kwargs):
+    def __init__(self, text='Select computer:', disabled_computers=False, path_to_root='../', **kwargs):
```

Comment:
Rather than making this a constructor argument, I'd propose to make this a `Bool` traitlet so that users can connect it to a checkbox etc.

I also propose to rename this to `allow_select_disabled` for clarity.

Reply:
I made it a trait, but I would still prefer to allow providing the value of the trait through the constructor argument. So I added the following:

```
        if 'allow_select_disabled' in kwargs:
            self.allow_select_disabled = kwargs['allow_select_disabled']
```